LDM-294.tex

```diff
@@ -0,0 +1,57 @@
+\documentclass[DM,lsstdraft,SDP,toc]{lsstdoc}
+%Replace the CUx here and all \CU will resolve below.
+
+%\usepackage{glossaries}
+%\makeglossaries
+%\setacronymstyle{long-short}
+%\newacronym{svm}{SVM}{support vector machine}
+%\newacronym{dm}{DM}{Data Management}
+%\usepackage{gwp}
+%\usepackage{risk}
+
+\setDocChangeRecord{%
+\addtohist{1}{2004-06-23}{Initial version} {JK}
+\addtohist{2}{2015-03-11}{Updated with new RFC process, realignment of TCT, SAT, DMLT - other versions in between}{JK}
+\addtohist{2}{2017-01-09}{Update in TeX}{WOM,MJ}
+	}
+
+\begin{document}
+
+\title[DM PMP]{Data Management Organization and Management }
+
+\author   {William O'Mullane, Mario Juric, Jeffrey P Kantor,  Tim Axelrod,  Roberta Allsman}                % the author(s) \setDocApprove  {LSST Project Oeefice}              % approval by ...
+\setDocRef      {LDM-294} % the reference code
+\setDocDate     {\today}              % the date of the issue
+
+%
+% a short abstract
+%
+\setDocAbstract {This is the DM plan updated from the v2 of 2014.
+   It covers the organisation and management of DM for LSST.}
+
+%
+% the title page
+%
+\maketitle
+
+%\printnoidxglossaries
+%
+% It's all yours from here on
+%
+\input{intro}
+\input{dmorg}
+\input{dmwbs}
+\input{dmproducts}
+\input{dmroles}
+\input{leadtutes}
+\input{dmgroups}
+\input{devprocess}
+\input{probman}
+
+\appendix
+\section{DM Product List \label{sect:prodlist}}
+\input{productlist}
+\input{DDMP}
```

Comment:
If this is in an appendix does it mean that the policy is not under change control? Is it even policy?